attestation/attestation.go

```diff
@@ -22,6 +22,8 @@ import (
 	"github.com/google/go-tspi/tspiconst"
 )
 
+var wellKnown [20]byte
```

Comment:
I know you're just moving this, but since it's global now can you rename it to something more descriptive. E.g. 'wellKnownSecret' or whatever it's being used as. Also add a comment